docs/unifiedpush/push-message-format/windows-document-format.asciidoc

```diff
@@ -0,0 +1,77 @@
+---
+layout: basic
+title: UnifiedPush Server - Windows message format
+---
+
+Windows message format
+----------------------
+
+By adding a windows section to your message you are able to use the advanced formatting for push messages on windows devices. Have a look at the [windows message catelog](https://msdn.microsoft.com/en-us/library/windows/apps/hh761491.aspx) to see what is posible. Be sure to have your application pinned to the start screen to see these notifications.
+
+Toast
+~~~~~
+Windows supports [toast messages](https://msdn.microsoft.com/en-us/library/windows/apps/hh761494.aspx) that have multiple lines of text, the first will be displayed bold. In order to keep the message as uniform as possible the first line will be the value of the message alert e.g.
+
+```json
+"message": {
+    "alert": "First line bold",
+    "windows": {
+      "type": "toast",
+      "toastType": "ToastText02",
+      "textFields": ["additional text"]
+    }
+}
+```
+This eanbles you to send these message to different platforms only adding extra information for windows users. MPNS does not support different toastTypes and a maximum of 2 lines of text additional lines are ignored.
```

Comment:
`enables` `messages` or replace `these` with `this`, `only by adding` , not sure for `2 lines of text additional lines`